java/daikon/chicory/DeclWriter.java

```diff
@@ -714,6 +480,115 @@ private String out_name(Enum<?> e) {
     return e.name().toLowerCase();
   }
 
+  /**
+   * Output most of the decl file information for a single variable.
+   *
+   * @param parent parent of var in the variable tree
+   * @param var variable whose values are to be output
+   * @param compare_ppt ppt with compare value if comparability-file present, null otherwise
+   * @param callback object containing getComparability method
+   */
+  public void print_decl(
```

Comment:
It's considered good Java style to use camelCase rather than snake_case for identifiers.  We don't have to go back and change all the old ones, but let's use camelCase for any newly-introduced identifiers.  I see you made some changes like this, such as changing print_decl_class to printDeclClass (thanks!), but you introduced other variable names in snake_case, such as creating a new print_decl method in that same class.  Could you change them?